### ISSUE TITLE
Some Questions

### ISSUE THREAD

**Issue by ghost** [rozbo/MyO9K#1]
Hello, I'm wondering if you are planing crack all other paid assemblies?

If not can u pm me small tutorial of bypassing ensage anti-dump module?
So I can do it myself.

Thank u very much.

**Comment by ohroy** [comment]
yes I had crack all paid assemblies and found a way to dump all module.
but I've been busy this time. I'll take the time to write the course later , then  I will release all the theories and tools.

**Comment by ghost** [comment]
Thank you, I appreciate it.

**Comment by ghost** [comment]
Just wondering Any eta on it?

**Comment by ghost** [comment]
*Bump*
